t/templates/_tag.tt

```diff
@@ -0,0 +1,8 @@
+<section>
```

Comment:
I would rather that this file just be a [% BLOCK %] section in tags.tt, the only template that invokes it.

Using a sub-template file like this isn't bad at all, but making it into a TT-block would be more consistent with the existing example templates.